.github/workflows/CI.yml

```diff
@@ -953,29 +953,24 @@ jobs:
       - name: Setup Dependencies Windows
         uses: msys2/setup-msys2@v2
         with:
+          msystem: ucrt64
           update: true
           install: >-
-            base-devel
-            diffutils
             doxygen
             git
-            make
-            mingw-w64-x86_64-binutils
-            mingw-w64-x86_64-boost
-            mingw-w64-x86_64-cmake
-            mingw-w64-x86_64-curl
-            mingw-w64-x86_64-graphviz
-            mingw-w64-x86_64-miniupnpc
-            mingw-w64-x86_64-nlohmann-json
-            mingw-w64-x86_64-nodejs
-            mingw-w64-x86_64-nsis
-            mingw-w64-x86_64-onevpl
-            mingw-w64-x86_64-openssl
-            mingw-w64-x86_64-opus
-            mingw-w64-x86_64-toolchain
-            nasm
-            wget
-            yasm
+            mingw-w64-ucrt-x86_64-boost
+            mingw-w64-ucrt-x86_64-cmake
+            mingw-w64-ucrt-x86_64-cppwinrt
+            mingw-w64-ucrt-x86_64-curl
+            mingw-w64-ucrt-x86_64-graphviz
+            mingw-w64-ucrt-x86_64-miniupnpc
+            mingw-w64-ucrt-x86_64-nlohmann-json
+            mingw-w64-ucrt-x86_64-nodejs
+            mingw-w64-ucrt-x86_64-nsis
```

Comment:
This one is missing from the docs.

Is it actually needed?

Reply:
It's here because before moving to ucrt64, CI installed the package `mingw-w64-x86_64-nsis`. I think it's used to build the installer bundle, so someone building it locally wouldn't need it but CI might.

Reply:
Makes sense, let's add that to the build docs as well. After that, I think this is okay to merge.